app/src/main/kotlin/maru/app/MaruAppFactory.kt

```diff
@@ -121,14 +121,7 @@ class MaruAppFactory {
         }
 
     val qbftFork = beaconGenesisConfig.getForkByConfigType(QbftConsensusConfig::class)
-    val qbftForkTimestamp = qbftFork.timestampSeconds.toULong()
     val qbftConfig = qbftFork.configuration as QbftConsensusConfig
-    BeaconChainInitialization(
```

Comment:
This constructor was mis-placed before given that it would be called before at line 120?

Reply:
That's my understanding, yes. I think we were trying to initialized the Beacon chain 2 times, but the second time it was just making sure that the state is initialized